docs/user-guide/install-customization.md

```diff
@@ -105,6 +108,42 @@ curl \
 curl --header "Authorization: Bearer $TOKEN" "http://$ASSISTED_SERVICE_IP:$ASSISTED_SERVICE_PORT/api/assisted-install/v2/clusters/$CLUSTER_ID/install-config"
 ```
 
+### Exclude one or more optional components (capabilities)
+
+Since [OpenShift 4.12](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md#resource-management), it is possible to customize the install config to disable some components.
+
+The features (`baremetal-operator`, `marketplace`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `openshift-samples-content`) can be disabled by setting the `baselineCapabilitySet` and `addittionalEnabledCapabilities` parameters in the `install-config.yaml`.
+
+To perform the customization, spec for capabilities must be adjusted, for example:
+
+```yaml
+capabilities:
+  baselineCapabilitySet: None
+  additionalEnabledCapabilities:
+    - openshift-samples
+```
+
+The `CVO`, will then calculate the effective status, that means, based on the [axioms](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md#axioms), CVO will allow to have them disabled at install time or still install them based on requirements, etc. Note that axioms will help in day-2 behaviors, for an installation, it will install the components not excluded or the required ones for dependencies.
+
+#### Patching the install config
+
+As with the parent example, install config can be patched with:
+
+```sh
+curl \
+    --header "Content-Type: application/json" \
+    --header "Authorization: Bearer $TOKEN" \
+    --request PATCH \
+    --data '"{\"controlPlane\":{\"capabilities\": {\"baselineCapabilitySet\": \"None\", \"additionalEnabledCapabilities\": [\"openshift-samples\"]}}"' \
```

Comment:
an extra } is missing from the stringified json, it should be
```
'"{\"controlPlane\":{\"capabilities\": {\"baselineCapabilitySet\": \"None\", \"additionalEnabledCapabilities\": [\"openshift-samples\"]}}}"'```

Reply:
changed